gammapy/visualization/utils.py

```diff
@@ -20,6 +23,69 @@
 }
 
 
+def plot_rgb(map_, energy_edges=None, ax=None, **kwargs):
```

Comment:
This function is mixing the plotting with the resampling. I think it would be better to split this. I.e. require  a map with three bins in z-axis in this function and explain in the docs how to use `resample_axis` to create a 3-bin axis from something that has more

Reply:
E.g. the API would be: `plot_rgb(map_, axis="energy", ax=None, **kwargs)` and inside you do a check that `map_.axes[axis].nbins == 3`